docs/components/operate/userguide/delete-finished-instances.md

```diff
@@ -1,46 +1,51 @@
 ---
 id: delete-finished-instances
 title: Delete finished instances
+description: "Let's delete a canceled or completed process instance."
 ---
 
-A finished process instance, meaning a canceled or a completed process instance, can be deleted from instances page or instance detail page.
+A finished process instance, meaning a canceled or a completed process instance, can be deleted from the **Instances** page or instance detail page.
 
-## Delete process instance from instances page
+## Delete process instance from Instances page
 
-1. On instances page, apply **Finished Instances** filter
+To delete a process instance from the **Instances** page, take the following steps:
+
+1. On the **Instances** page, apply the **Finished Instances** filter.
 
 ![operate-view-finished-instances](./img/operate-instances-finished-instances_light.png)
 
-2. Click delete icon on any process instance you want to delete
+1. Click the trash can icon on any process instance you want to delete.
 
 ![operate-perform-delete-operation](./img/operate-instances-click-delete-operation_light.png)
 
-3. Confirm the delete operation by clicking 'Delete'
+1. Confirm the delete operation by clicking **Delete**.
 
 ![operate-confirm-delete-operation](./img/operate-instances-delete-operation-confirm_light.png)
 
-4. Delete operation can be seen in operations panel
+4. In the **Operations** panel on the right side of the screen, view the deleted process instance.
 
 ![operate-view-delete-operation](./img/operate-operations-panel-delete-operation_light.png)
 
 ## Delete process instance from instance detail page
 
-1. On instances page, apply **Finished Instances** filter
+1. On the **Instances** page, apply the **Finished Instances** filter.
 
 ![operate-view-finished-instances-instance-detail](./img/operate-instance-detail-finished-instances_light.png)
 
-2. Navigate to instance detail page, by clicking the id of the process instance you want to delete
+2. Navigate to the instance detail page by clicking the **Instance id** of the process instance you want to delete.
 
 ![operate-navigate-finished-instance-detail](./img/operate-instance-detail-finished-instances-navigate_light.png)
 
-3. Click the delete icon
+3. Click the delete icon.
 
 ![operate-instance-detail-perform-delete](./img/operate-finished-instance-detail_light.png)
 
-4. Confirm the delete operation by clicking 'Delete'
+1. Confirm the delete operation by clicking **Delete**.
 
 ![operate-instance-detail-confirm-delete-operation](./img/operate-instance-detail-delete-operation-confirm_light.png)
 
-5. Process instance is deleted
+The process instance is now deleted.
 
 ![operate-instance-deleted-notification](./img/operate-instance-deleted-notification_light.png)
+
+[//]:# (Is there any chance of getting this process instance back? If not, we should probably note a warning.)
```

Comment:
@santanaluiz Is this a hard delete, dropped from the database entirely? Could CCSM users retrieve it from a database?

Reply:
AFAIK, we can't have access to it again. It is a hard delete, mostly for GDPR purposes